src/you_get/common.py

```diff
@@ -682,19 +690,21 @@ def __init__(self, total_size, total_pieces = 1):
     def update(self):
         self.displayed = True
         bar_size = self.bar_size
-        percent = round(self.received * 100 / self.total_size, 1)
-        if percent >= 100:
+        percent_str = round_str(self.received * 100 / self.total_size, 1)
+        if self.received == self.total_size:
             percent = 100
```

Comment:
For aesthetic reasons, better set `percent_str` to 100 too. (instead of the string `"100.0"`)

```py
percent_str = percent = 100
```